src/components/BusyIndicator.js

```diff
@@ -158,6 +182,24 @@ export default class BusyIndicator extends React.Component<BusyIndicatorDefaultP
         ? <div style={mergedMessageStyle}>{message}</div>
         : '';
 
+      if (type === 'spinnyCentered') {
+        return (
+          <div className="attivio-spinner">
+            {this.renderBusySymbol()}
+            <br />
```

Comment:
If there's no `messageToShow`, you probably don't want to add these `<br>`s and the message at all.